### PR TITLE
Use core instead of std and hygiene refactor

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by sylv256** [comment]
> Update lib.rs

most descriptive commit message

File: oop-macro/src/lib.rs [hunk 1]
```diff
@@ -33,7 +33,7 @@ pub fn extend(attr: TokenStream, mut item: TokenStream) -> TokenStream {
 	let field_name = args.field_name;
 	item.extend::<TokenStream>(
 		quote! {
-			use core::ops::{Deref, DerefMut};
+			use ::core::ops::{Deref, DerefMut};
```
Comment (sylv256) [inline oop-macro/src/lib.rs]: still a `use`. i'll fix it

**Review by sylv256 — CHANGES_REQUESTED** [review]
also we gotta fix the macro hygiene

File: oop-macro/src/lib.rs (hunk 1 above)
Comment (ghost) [reply oop-macro/src/lib.rs]: I removed it in a later commit, see the updated commit please

**Review by sylv256 — APPROVED** [review]
LGTM

**Pull request merged**